pkg/apis/kops/cluster.go

```diff
@@ -286,6 +286,7 @@ type LoadBalancerAccessSpec struct {
 	Type                     LoadBalancerType `json:"type,omitempty"`
 	IdleTimeoutSeconds       *int64           `json:"idleTimeoutSeconds,omitempty"`
 	AdditionalSecurityGroups []string         `json:"additionalSecurityGroups,omitempty"`
+	UseApiInternal           bool             `json:"useApiInternal,omitempty"`
```

Comment:
Naming nit: it isn't very obvious (to me) what this does from the name.  How about `useForInternalApi` or `inClusterApi` or `useInsideCluster` ?

(And we have to take another pass to add this to v1alpha1 anyway ... :-) )

Reply:
yep fair enough, I wasn't too keen on the name myself, I think `useForInternalApi` is clearer